contracts/interfaces/IInterchainToken.sol

```diff
@@ -2,12 +2,20 @@
 
 pragma solidity ^0.8.0;
 
-import { IERC20 } from '@axelar-network/axelar-gmp-sdk-solidity/contracts/interfaces/IERC20.sol';
+import { ITokenManager } from './ITokenManager.sol';
 
 /**
  * @dev Interface of the ERC20 standard as defined in the EIP.
  */
-interface IInterchainToken is IERC20 {
+interface IInterchainToken {
+
+    /**
+     * @notice Getter for the tokenManager used for this token.
+     * @dev Needs to be overwitten.
+     * @return tokenManager the TokenManager called to facilitate cross chain transfers.
+     */
+    function getTokenManager() external view returns (ITokenManager tokenManager);
```

Comment:
```suggestion
    function tokenManager() external view returns (ITokenManager tokenManager);
```
We're inconsistent with `get*` vs the name itself for getters in different places. The latter seems to be a common practice.

Reply:
A broader question here. `IInterchainToken` is supposed to represent the interchain token standard. This doesn't technically depend on having a token manager. Should this be added to the standardized token interface instead?